src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -569,6 +569,7 @@ private static bool IsStandardImplicitConversionFromExpression(ConversionKind ki
                 case ConversionKind.ImplicitTupleLiteral:
                 case ConversionKind.StackAllocToPointerType:
                 case ConversionKind.StackAllocToSpanType:
+                case ConversionKind.InterpolatedStringHandler:
```

Comment:
>case ConversionKind.InterpolatedStringHandler: [](http://example.com/codeflow?start=16&length=46)

Is there a relevant portion of the spec to support this change?

Reply:
After staring at the spec and thinking about it more, I think the answer is no, there isn't, and I need a different change here.

Reply:
Actually, I'm going to send an email to LDM to follow up on this one, and I'll update the spec if we decide that it should be a standard implicit conversion.